Python/Extract_Contents_of_Zip/extractScript.py

```diff
@@ -0,0 +1,14 @@
+#import pyshorteners library
+import pyshorteners
+
```

Comment:
you need to remove this URL shortener script from this pr.

Reply:
Ok